kingfisher_scrapy/spiders/peru_compras_releases.py

```diff
@@ -0,0 +1,43 @@
+import scrapy
+
+from kingfisher_scrapy.base_spider import SimpleSpider
+from kingfisher_scrapy.util import handle_http_error, parameters
+
+
+class PeruComprasReleases(SimpleSpider):
+    """
+    Domain
+      Peru Compras - Framework Agreements
+    Spider arguments
+      from_date
+        Download only data from this date onward (YYYY-MM-DD format). Defaults to '2017-01-01'.
+      until_date
+        Download only data until this date (YYYY-MM-DD format). Defaults to today.
+    """
+
+    name = 'peru_compras_releases'
```

Comment:
```suggestion
    name = 'peru_compras'
```
As this is the only spider for Peru, I think that we can name it just "peru_compras" (same for the class name)